src/btree/bt_vrfy.c

```diff
@@ -269,8 +269,7 @@ __wt_verify(WT_SESSION_IMPL *session, const char *cfg[])
             WT_ERR(__wt_btree_tree_open(session, root_addr, root_addr_size));
 
             if (WT_VRFY_DUMP(vs))
-                WT_ERR(__wt_msg(session, "- Root: %s\n\t> addr: %s",
-                  __wt_addr_string(session, root_addr, root_addr_size, vs->tmp1),
+                WT_ERR(__wt_msg(session, "- %p: root\n\t> addr: %s", (void *)root_addr,
                   __wt_addr_string(session, root_addr, root_addr_size, vs->tmp1)));
```

Comment:
I suppose we could keep this as it originally was - 

```
                WT_ERR(__wt_msg(session, "Root: %s",
                  __wt_addr_string(session, root_addr, root_addr_size, vs->tmp1),
```
and parse it accordingly. I just noticed it now that the next node is the root node.

Reply:
i thought it would be better to keep it consistent and have 
`- ref: page_type`